docs/src/installation.md

```diff
@@ -45,7 +45,7 @@ Solver support in Julia is currently provided by writing a solver-specific packa
 | ''                                                                               | [CoinOptServices.jl](https://github.com/JuliaOpt/CoinOptServices.jl)            | `OsilCouenneSolver()`     |  ''         |       |       |       |       |       |       |
 | [CPLEX](http://www-01.ibm.com/software/commerce/optimization/cplex-optimizer/)   | [CPLEX.jl](https://github.com/JuliaOpt/CPLEX.jl)                                | `CplexSolver()`           |  Comm.      | X     | X     | X     |       |       |       |
 | [ECOS](https://github.com/ifa-ethz/ecos)                                         | [ECOS.jl](https://github.com/JuliaOpt/ECOS.jl)                                  | `ECOSSolver()`            |  GPL        | X     | X     |       |       |       |       |
-| [FICO Xpress](http://www.fico.com/en/products/fico-xpress-optimization-suite)    | [Xpress.jl](https://github.com/JuliaOpt/Xpress.jl)                              | `XpressSolver()`          |  Comm.      | X     | X     | X     |       |       |       |
+| [FICO Xpress](http://www.fico.com/en/products/fico-xpress-optimization-suite)    | [Xpress.jl](https://github.com/JuliaOpt/Xpress.jl)                              | `Xpress.XpressSolver()`          |  Comm.      | X     | X     | X     |       |       |       |
```

Comment:
Have you tested that the alignment is correct and the documentation compiles correctly?

Reply:
alignment looks good on #1104

Reply:
#1104 does not test this. You have to build a local copy of the docs by following https://github.com/JuliaOpt/JuMP.jl/tree/master/docs#jumpjl-documentation-readme